docs/api/IoT.js-API-File-System.md

```diff
@@ -6,297 +6,588 @@ The following shows fs module APIs available for each platform.
+
+
+
+### `stats.isDirectory()`
+* Returns: {boolean}
+
+Returns true if stated file is a directory.
+
+
+### `stats.isFile()`
+* Returns: {boolean}
+
+Returns true if stated file is a file.
 
+**Example**
+
+```js
+var assert = require('assert');
+var fs = require('fs');
 
-## Module Functions
+fs.stat('test.txt', function(err, stat) {
+  if (err) {
+    throw err;
+  }
+  assert.equal(stat.isFile(), true);
+  assert.equal(stat.isDirectory(), false);
+});
+```
+
+
+## Class methods
+
+### Class method: fs.close(fd, callback)
+* `fd` {integer} File descriptor.
+* `callback` {Function(err: Error | null)} Callback function.
+
+Closes the file of `fd` asynchronously.
+
+**Example**
 
-### `fs.open(path, flags[, mode], callback)`
-* `path <String>` - file path to be opened.
-* `flags <String>` - open flags.
-* `mode <Number>` Default: `0666` - permission mode.
-* `callback <Function(err: Error | null, fd: Number)>`
+```js
+var fs = require('fs');
+
+fs.open('test.txt', 'r', function(err, fd) {
+  if (err) {
+    throw err;
+  }
+  // do something
+  fs.close(fd, function(err) {
+    if (err) {
+      throw err;
+    }
+  });
+});
+```
+
+
+### Class method: fs.closeSync(fd)
+* `fd` {integer} File descriptor.
+
+Closes the file of `fd` synchronously.
+
+**Example**
+
+```js
+var fs = require('fs');
+
+var fd = fs.openSync('test.txt', 'r');
+// do something
+fs.closeSync(fd);
+```
+
+
+### Class method: fs.exists(path, callback)
+* `path` {string} File path to be checked.
+* `callback` {Function(exists: boolean)}
+
+Checks the file specified by `path` exists asynchronously.
+
+**Example**
+
+```js
+var assert = require('assert');
+var fs = require('fs');
+
+fs.exists('test.txt', function(exists) {
+  assert.equal(exists, true);
+});
+```
+
+
+### Class method: fs.existsSync(path)
+* `path` {string} File path to be checked.
+* Returns: {boolean} True if the file exists, otherwise false.
+
+Checks the file specified by `path` exists synchronously.
+
+```js
+var assert = require('assert');
+var fs = require('fs');
+
+var result = fs.existsSync('test.txt');
+assert.equal(result, true);
+```
+
+
+### Class method: fs.mkdir(path[, mode], callback)
+* `path` {string} Path of the directory to be created.
+* `mode` {number} Permission mode. **Default:** `0777`
+* `callback` {Function(err: Error | null)}
+
+Creates the directory specified by `path` asynchronously.
+
+**Example**
+
+```js
+var fs = require('fs');
+
+fs.mkdir('testdir', function(err) {
+  if (err) {
+    throw err;
+  }
+});
+```
+
+
+### Class method: fs.mkdirSync(path[, mode])
+* `path` {string} Path of the directory to be created.
+* `mode` {number} Permission mode. **Default:** `0777`
+
+Creates the directory specified by `path` synchronously.
+
+**Example**
+
+```js
+var fs = require('fs');
+
+fs.mkdirSync('testdir')
+```
+
+
+### Class method: fs.open(path, flags[, mode], callback)
+* `path` {string} File path to be opened.
+* `flags` {string} Open flags.
```

Comment:
no explanation for `flags`. Please refer to https://github.com/nodejs/node/blob/master/doc/api/fs.md#fsopenpath-flags-mode-callback